redis/src/commands/json.rs

```diff
@@ -0,0 +1,201 @@
+// can't use rustfmt here because it screws up the file.
+#![cfg_attr(rustfmt, rustfmt_skip)]
+use crate::cmd::{cmd, Cmd};
+use crate::connection::ConnectionLike;
+use crate::pipeline::Pipeline;
+use crate::types::{FromRedisValue, RedisResult, ToRedisArgs};
+use crate::{RedisError, ErrorKind};
+
+#[cfg(feature = "cluster")]
+use crate::commands::ClusterPipeline;
+
+use serde::ser::Serialize;
+
+impl From<serde_json::Error> for RedisError {
+	fn from(serde_err: serde_json::Error) -> RedisError {
+		RedisError::from((
+			ErrorKind::Serialize,
+			"Serialization Error",
+			format!("{}", serde_err)
+		))
+	}
+}
+
+implement_json_commands! {
+    'a
+	
+	/// Append the JSON `value` to the array at `path` after the last element in it.
+	fn json_arr_append<K: ToRedisArgs, P: ToRedisArgs, V: Serialize>(key: K, path: P, value: &'a V) {
+		Ok::<Cmd, RedisError>(
+			cmd("JSON.ARRAPPEND")
+				.arg_take(key)
+				.arg_take(path)
+				.arg_take(serde_json::to_string(value)?)
+		)
+	}
+
+	/// Index array at `path`, returns first occurance of `value`
+	/// Pass `None` to `start_stop` to default it to `0, 0` (the redis default in this command)
```

Comment:
```suggestion
	/// Index array at `path`, returns first occurance of `value`
	///
	/// Pass `None` to `start_stop` to default it to `0, 0` (the Redis default in this command).
```

Reply:
I just realised I forgot to add `start` and `stop` to this command, will fix